core/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -243,19 +242,10 @@ WorkerServiceHandler getWorkerServiceHandler() {
     return mWorkerServiceHandler;
   }
 
-  private void login() throws IOException {
-    String workerKeytabFile = mTachyonConf.get(Constants.WORKER_KEYTAB_KEY, null);
-    String workerPrincipal = mTachyonConf.get(Constants.WORKER_PRINCIPAL_KEY, null);
-    if (workerKeytabFile == null || workerPrincipal == null) {
-      return;
-    }
+  private void connect() throws IOException {
```

Comment:
I'll rename this to connectToUFS in the next iteration to keep it consistent with that we did on the master side.